packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -679,6 +760,23 @@ class ContainerLayer extends Layer {
     assert(transform != null);
   }
 
+  /// Returns the descendants of this layer in depth first order.
```

Comment:
Document that it may return null in certain cases?

Reply:
Changed this to not return null.